packages/framework-core/src/decorators/global-event-handler.ts

```diff
@@ -0,0 +1,24 @@
+import { EventHandlerInterface, BoosterConfig } from '@boostercloud/framework-types'
+import { Booster } from '../booster'
+
+export const GLOBAL_EVENT_HANDLER = 'GLOBAL_EVENT_HANDLER'
+
+export function GlobalEventHandler(): <TEventHandler extends EventHandlerInterface>(
+  eventHandlerClass: TEventHandler
+) => void {
+  return (eventHandlerClass) => registerGlobalEventHandler(eventHandlerClass)
+}
+
+function registerGlobalEventHandler<TEventHandler extends EventHandlerInterface>(
+  eventHandlerClass: TEventHandler
+): void {
+  Booster.configureCurrentEnv((config: BoosterConfig): void => {
+    const registeredEventHandlers = config.eventHandlers[GLOBAL_EVENT_HANDLER] || []
+    if (registeredEventHandlers.length > 0) {
+      throw new Error(`A global event handler is already registered.
+        If you think that this is an error, try performing a clean build.`)
+    }
+    registeredEventHandlers.push(eventHandlerClass)
+    config.eventHandlers[GLOBAL_EVENT_HANDLER] = registeredEventHandlers // todo
```

Comment:
This code prevents having more than one global event handler in the application, and I think we shouldn't do that. It can be very useful to have different global error handler for different purposes (for example, one to send logs to some third party, another one to do some other tracking, etc.).

By allowing that, then this method can be deleted and reuse the existing one in `event-handler.ts`. See next comment.